### ISSUE TITLE
[usability]: Unable to match 8.6 as a known SolidFire Element OS version

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#14]
v1.3.0.124 gives this error on attempt to connect to an Element OS v8.6 cluster
```
Credential
Connect-SFCluster : Unable to match 8.6 as a known SolidFire Element OS version.
At line:1 char:1
+ Connect-SFCluster 10.128.37.90
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OpenError: (:) [Connect-SFCluster], VersionException
    + FullyQualifiedErrorId : ConnectionError,SolidFire.Connection.ConnectSFCluster
```

`-versionapi 8` does work, but I'm wondering if we could use the part of version info before the first separator `.` so in case vX.Y.Z is detected, use version X.
If the API version isn't explicitly set by the client, it seems reasonable to fall back to the latest version available. Or maybe: if it's not set *and* if the client version is newer. 
Just an idea.

**Comment by ahaid** [comment]
@scaleoutsean I have a hotfix for this ready for you. I've tested it on my end. Would you mind trying it out before I release? Send me an email so I can send it to you. adam.haid@netapp.com

**Comment by ahaid** [comment]
This will be addressed in the 1.3.1 hotfix. I'll close the issue when it is released.

**Comment by scaleoutsean** [comment]
Thank you - the hotfix version works well without `-versionapi`.
I'll let you close this when you think is better - now or once the new version is out.

**Comment by scaleoutsean** [comment]
Today I used the freshly released `v1.3.1.4` with another cluster running v8.6.0.25 - also no issues, so closing this.
